doc/cli_certificate.md

```diff
@@ -0,0 +1,27 @@
+# Certificate
+
+Tooling for offline transaction creation
+
+## Builder
+
+Builds a signed certificate.
+
+The process can be split into steps on the first step certificate
+is created. 
+```
+jcli certificate new stake-pool-registration \
+  --vrf-key <vrf-public-key> --kes-key <kes-public-key> \
+  [--owner <owner-public-key>] \
```

Comment:
so, `0, 1 or more` ? Don't we need at least *one* owner ?

Reply:
yes currently it's possible to create a cert without owners, should I fix that?

Reply:
No, not yet. We might come back to it though